SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -772,6 +772,14 @@ void ActorSystem::UpdateShadowData(const Worker_EntityId EntityId) const
 {
 	USpatialActorChannel* ActorChannel = NetDriver->GetActorChannelByEntityId(EntityId);
 	ActorChannel->UpdateShadowData();
+	const ComponentData* InterestBucketComponent =
```

Comment:
Interest buckets need to be tracked, lest a server add another tag after getting handover.